MaterialComponents.podspec

```diff
@@ -28,171 +28,195 @@ Pod::Spec.new do |mdc|
   # # Template subspec
   #
   #  mdc.subspec "ComponentName" do |component|
-  #    component.public_header_files = "components/#{component.base_name}/src/*.h"
-  #    component.source_files = "components/#{component.base_name}/src/*.{h,m}", "components/#{component.base_name}/src/private/*.{h,m}"
+  #.   component_name="#{component.base_name}"
   #
-  #    # Only if you have a resource bundle
-  #    component.resources = ["components/#{component.base_name}/src/Material#{component.base_name}.bundle"]
+  #    component.public_header_files = "components/#{component_name}/src/*.h"
+  #    component.source_files = "components/#{component_name}/src/*.{h,m}", "components/#{component_name}/src/private/*.{h,m}"
   #
+  #    # Only if you have a resource bundle
+  #    component.resources = ["components/#{component_name}/src/Material#{component_name}.bundle"]
   #  end
   #
 
-  mdc.subspec "ActivityIndicator" do |component|
-    component.subspec "Component" do |spec|
-      spec.ios.deployment_target = '8.0'
-      spec.public_header_files = "components/#{component.base_name}/src/*.h"
-      spec.source_files = "components/#{component.base_name}/src/*.{h,m}", "components/#{component.base_name}/src/private/*.{h,m}"
-      spec.resources = ["components/#{component.base_name}/src/Material#{component.base_name}.bundle"]
+  mdc.subspec "ActivityIndicator" do |spec|
+    component_name="#{spec.base_name}"
+
+    spec.subspec "Component" do |component|
```

Comment:
why does swapping component for spec variables make it easier to understand or harder to make mistake?